src/ducks/transactions/TriggerErrorCard.spec.jsx

```diff
@@ -0,0 +1,32 @@
+/* global mount */
```

Comment:
Why don't you get it from enzyme?

Reply:
I think it's everywhere in Banks. mount is global. It is not very useful since we have to put this comment... I'll make a pass on test files to replace with an import.